orbit/pkg/packaging/macos_templates.go

```diff
@@ -37,8 +37,17 @@ ln -sf /var/lib/orbit/bin/orbit/macos/{{.OrbitChannel}}/orbit /var/lib/orbit/bin
 ln -sf /var/lib/orbit/bin/orbit/orbit /usr/local/bin/orbit
 
 {{ if .StartService -}}
-launchctl unload /Library/LaunchDaemons/com.fleetdm.orbit.plist
-launchctl load -w /Library/LaunchDaemons/com.fleetdm.orbit.plist
+DAEMON_LABEL="com.fleetdm.orbit"
+DAEMON_PLIST="/Library/LaunchDaemons/${DAEMON_LABEL}.plist"
+
+# Remove any pre-existing version of the config
+launchctl bootout system "${DAEMON_PLIST}"
```

Comment:
Nitpick: Looks like `launchctl bootout` and `launchctl bootstrap` also support the service-target syntax, maybe for consistency you could use them:
```
launchctl bootout "system/${DAEMON_LABEL}"
...
launchctl bootstrap "system/${DAEMON_LABEL}"
```

Reply:
I'll test this out and verify the functionality still works as expected. My assumption is that it's a bootstrap problem where the `system/${DAEMON_LABEL}` syntax doesn't work because the launchd system doesn't yet know about the daemon.

Reply:
I've tried a few different permutations of this, but it appears that `launchctl bootstrap` requires a service-path instead of a service-target.

> Usage: launchctl bootstrap <domain-target> [service-path, service-path2, ...]

`launchctl bootout "system/${DAEMON_LABEL}"` does appear to work as expected, so I'll be pushing that commit